src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -894,10 +894,15 @@ def __create_truncated_status_file(self, status_file_size_in_bytes, complete_sta
                 self.__start_truncation_process(self.__assessment_patches_copy, self.__installation_patches_copy, max_allowed_patches_size_in_bytes, low_pri_index)
 
             if len(self.__assessment_patches_removed) > 0:
+                assessment_tombstone_list = self.__create_assessment_tombstones_by_classification(self.__assessment_patches_removed)
+                patches_retained_in_assessment.extend(assessment_tombstone_list)     # Add assessment tombstone list
+
                 self.composite_logger.log_verbose("Recomposing truncated status payload: [Substatus={0}]".format(Constants.PATCH_ASSESSMENT_SUMMARY))
                 truncated_status_file = self.__recompose_truncated_status_file(truncated_status_file=truncated_status_file, truncated_patches=patches_retained_in_assessment, count_total_errors=self.__assessment_total_error_count, substatus_message=self.__assessment_substatus_msg_copy, substatus_status=assessment_substatus_status, substatus_index=assessment_substatus_index)
 
             if len(self.__installation_patches_removed) > 0:
+                # Todo need further requirements to decompose installation tombstone by classifications
```

Comment:
I added here as reminder because the installation tombstone definitely needs to be details just as you said the classification and package should be descrptive, should I remove it, then i will create a ado item in our backlog to track it?

Reply:
removed, done